core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/ext/CustomLinkResolver.java

```diff
@@ -0,0 +1,12 @@
+package com.exadel.etoolbox.linkinspector.core.services.ext;
+
+import com.exadel.etoolbox.linkinspector.api.service.LinkTypeProvider;
+import com.exadel.etoolbox.linkinspector.core.models.Link;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public interface CustomLinkResolver {
+    List<LinkTypeProvider> getLinkTypeProviders();
```

Comment:
Do we need this method? It seems like providers are only used within the "resolver" service implementation